autogen/agentchat/conversable_agent.py

```diff
@@ -632,10 +665,12 @@ def generate_oai_reply(
             context=messages[-1].pop("context", None), messages=self._oai_system_message + messages
         )
 
-        # TODO: line 301, line 271 is converting messages to dict. Can be removed after ChatCompletionMessage_to_dict is merged.
+        # TODO: line 270, 298, 428, 479 are converting messages to dict. Can be removed after ChatCompletionMessage_to_dict is merged.
```

Comment:
ChatCompletionMessage_to_dict was replace with response.mode_dump, so all messages will be converted to dict from generate_oai_reply. So this can be removed?

Reply:
yeah, this is referencing the message_to_dict method and its calls